server/flaskr/blueprints/auth.py

```
@@ -48,15 +50,41 @@ def login(email, password):
+        200,
+    )
+
+
+@auth_bp.post("/refresh")
+def refresh():
+    if "Refresh" in request.headers:
+        token = request.headers.get("Refresh", "")
+
+    try:
+        data = jwt.decode(token, current_app.config["SECRET_KEY"], algorithms=["HS256"])
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)